apps/suredbits-wallet/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.7"
+
+services:
+  web:
+    image: bitcoinscala/wallet-server-ui:0.0.1-af54042b-SNAPSHOT
+    user: "1000:1000"
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      WALLET_SERVER_API_URL: "http://${APP_SUREDBITS_WALLET_SERVER_IP}:9999/"
+      WALLET_SERVER_WS: "ws://${APP_SUREDBITS_WALLET_SERVER_IP}:19999/events"
+      TOR_PROXY: socks5://${TOR_PROXY_IP}:${TOR_PROXY_PORT}
+    ports:
+      - "${APP_SUREDBITS_WALLET_PORT}:3002"
+    networks:
+      default:
+        ipv4_address: $APP_SUREDBITS_WALLET_IP
+    depends_on: 
+      - walletserver
+  walletserver:
+    image: bitcoinscala/bitcoin-s-server:1.8.0-90-0c625346-SNAPSHOT
+    user: "1000:1000"
+    restart: on-failure
+    volumes:
+      - ${APP_DATA_DIR}/data/wallet:/home/bitcoin-s/.bitcoin-s
+    environment:
+      BITCOIN_S_NETWORK: $BITCOIN_NETWORK
+      BITCOIN_S_KEYMANAGER_ENTROPY: $APP_SEED  
+      BITCOIN_S_PROXY_ENABLED: "false"
+      BITCOIN_S_TOR_ENABLED: "false"
+      BITCOIN_S_TOR_PROVIDED: "true"
+      BITCOIN_S_DLCNODE_PROXY_ENABLED: "true"
+      BITCOIN_S_DLCNODE_PROXY_SOCKS5: "${TOR_PROXY_IP}:${TOR_PROXY_PORT}"
+      BITCOIN_S_DLCNODE_EXTERNAL_IP: $SUREDBITS_WALLET_P2P_HIDDEN_SERVICE
```

Comment:
`$SUREDBITS_WALLET_P2P_HIDDEN_SERVICE` should be the Tor hidden service for this container that `$APP_SUREDBITS_WALLET_P2P_PORT` is exposed at, right?

Reply:
Oh nice just realised you already added the hidden service and setup the env var yourself!